testing/test_jsonnet.py

```diff
@@ -38,18 +38,32 @@
 from kubeflow.testing import test_util
 from kubeflow.testing import util
 
+# We should test all files which end in .jsonnet or .libsonnet
+# except ksonnet prototype definitions - they require additional
```

Comment:
Aren't the files in prototypes jsonnet, and otherwise libsonnet?

Reply:
We cannot test the prototypes directly since they are not valid jsonnet. For example: ksonnet injects the `params` and the `env` variable into the prototype when a component is generated from that prototype.

The idea is that we want to test everything(jsonnet and libsonnet file) under the kubeflow/ directory unless it is a ksonnet prototype.

Reply:
I mean, is the following true:
.jsonnet --> ksonnet prototype, cannot test
.libsonnet --> valid, can test

I manually checked, seems to be the case?

Reply:
Yes. We shouldn't test any .jsonnet under .../prototypes/ since they are ksonnet prototypes.